### PR TITLE
pgBackRest containers cleanup and update

### PR DESCRIPTION
**Pull request by tjmoore4**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the new behavior (if this is a feature change)?**
The pgo-backrest and pgo-backrest-repo containers have been
moved to the Crunchy Containers project. As such, the associated
files are no longer needed in this repository.

Additionally, the references to these containers are now updated
to match the new naming convention being used, and the image
tag and prefix values are updated to reflect the new location
of the containers.

**Other information**:

### CODE REVIEW

**Review by jkatz — COMMENTED** [review]
Looks good so far :+1:

File: installers/ansible/roles/pgo-operator/files/pgo-configs/backrest-job.json [hunk 1]
```diff
@@ -31,14 +31,20 @@
...
                         {{.PgbackrestRestoreVolumeMounts}}
                     ],
                     "env": [{
                         "name": "COMMAND",
                         "value": "{{.Command}}"
                     }, {
+                        "name": "MODE",
+                        "value": "pgbackrest"
+                    },{
+                        "name": "CRUNCHY_DEBUG",
```
Comment (jmckulk) [inline installers/ansible/roles/pgo-operator/files/pgo-configs/backrest-job.json:45]: Debug needs to be removed

File: installers/ansible/roles/pgo-operator/files/pgo-configs/pgo-backrest-repo-template.json [hunk 2]
```diff
@@ -48,14 +48,21 @@
...
                         "protocol": "TCP"
                     }],
                     {{.ContainerResources }}
                     "env": [
                       {{.PgbackrestS3EnvVars}}
+                      {
+                        "name": "MODE",
+                        "value": "pgbackrest-repo"
+                      },{
+                        "name": "CRUNCHY_DEBUG",
```
Comment (jmckulk) [inline installers/ansible/roles/pgo-operator/files/pgo-configs/pgo-backrest-repo-template.json:63]: same here, remove debug

**Comment by jkatz** [comment]
Superseded by #2093

**Pull request closed without merging**